services/utilities.py

```diff
@@ -32,12 +32,15 @@
 def tag_image(image_path, limit=5):
     # Limit is set to 5 as default no of maximum tags
 
-    config = ConfigParser()
-    config.read('auth.ini')
+    # config = ConfigParser()
+    # config.read('auth.ini')
 
     # Configuring key to communicate with imagga API
-    api_key = config.get('imagga_credentials', 'api_key')
-    api_secret = config.get('imagga_credentials', 'api_secret')
+    # api_key = config.get('imagga_credentials', 'api_key')
+    # api_secret = config.get('imagga_credentials', 'api_secret')
+
+    api_key = 'acc_c3b083251e16281'
```

Comment:
@teeniv001 I request you to use the ```ConfigParser()``` to fetch the credentials from the auth.ini as hard-coding credentials is not a good practice.